v5/api/c/rotation.rst

```diff
@@ -234,7 +236,9 @@ Analogous to `pros::Rotation::get_velocity <../cpp/rotation.html#get-velocity>`_
 rotation_get_angle
 ------------------
 
-Get the Rotation Sensor's current angle in centidegrees (0-36000)
+Get the Rotation Sensor's current position in centidegrees. This means the absolute position
```

Comment:
Similar request for a tweak to the wording:
```
Get the Rotation Sensor's _absolute_ angle in centidegrees (0-36000). This angle is measured in relation to its 0 centidegree position when the sensor was last initialized or reset. The angle will roll over to 0 as the sensor rotates past +-36000 centidegrees.
```

Reply:
Is it last reset or is it some global 0 angle on the sensor? Not sure how reset survives restarts?

Reply:
If it survives restarts I'd say that explicitly.

Reply:
Also I think Jonathan's "rotates past +-36000" is not accurate. As much as I would love it to be a zero centered range I think it's (0,36000). 
So that part could be, 
"As it rotates below 0 it wraps around to 36000 and vice versa for above 36000; always returning the distance from 0 point irrespective of the path the sensor took to get there"

A bit wordy but hopefully more explicit

Reply:
So the reason why this is a little tricky to deal with wording is because since the rotation sensor is a Hall effect sensor, angle is maintained between resets. Position's initial value after reset is set to angle as well.

Reply:
Ahhh so it's technically distance from reset but not for the reason you might think.


My opinion is if the 2 "modes" are so confusing to need a paragraph to explain it properly take a paragraph. This is the source for someone to try to understand what the hell it does. I wouldn't have guessed get_position reset to get_angle instead of 0. Now that you say that I can think why, get_position is likely defined as get_angle + full_rotations * 36000,with reset only wiping full rotations, but that wasn't my educated guess.